action.yml

```diff
@@ -44,27 +44,55 @@ runs:
       with:
         detached: true
 
-    - name: Load Machine User Name
-      id: load_machine_user
-      env:
-        GH_TOKEN: ${{ inputs.token }}
-      shell: bash
+    - name: Check Pipelines Execute can run
+      if: ${{ startsWith(inputs.working_directory, '_new-account-request') }}
```

Comment:
I'm not sure if this is necessary anymore. I don't think the working directory will ever be `_new-account-requests`, because we always apply those from the mgmt account.